src/main/java/com/google/cloud/genomics/utils/grpc/VariantUtils.java

```diff
@@ -38,6 +39,28 @@
   public static final String GATK_NON_VARIANT_SEGMENT_ALT = "<NON_REF>";
 
   /**
+   * Determines whether a variant represents a multi-nucleotide change.
+   */
+  public static final Predicate<Variant> IS_MULTI_NUCLEOTIDE = new Predicate<Variant>() {
+    @Override
+    public boolean apply(Variant variant) {
+      int refSize = variant.getReferenceBases().length();
+      if (refSize < 1) {
+        return false;
+      } else if (refSize > 1) {
+        return true;
+      }
+      for (String alt : variant.getAlternateBasesList()) {
```

Comment:
Eh, what can I say, it's probably the first time I've seen it coded indirectly like this.  The first check will never exist as you need a reference to do any sort of analysis (i.e. definition of at least a point of reference :), so it probably should be removed one day with the rest of the code to reflect more directly the cases seen in (g)VCF files.  Thanks @deflaux for the clarification.

@hwangy Just for future reference, here's the definition for `REF`:

> REF - reference base(s): Each base must be one of A,C,G,T,N (case insensitive). Multiple bases are permitted.  The value in the POS field refers to the position of the first base in the String.  For simple insertions and deletions in which either the REF or one of the ALT alleles would otherwise be null/empty, the REF and ALT Strings must include the base before the event (which must be reflected in the POS field), unless the event occurs at position 1 on the contig in which case it must include the base after the event; this padding base is not required (although it is permitted) for e.g. complex substitutions or other events where all alleles have at
least one base represented in their Strings. If any of the ALT alleles is a symbolic allele (an angle-bracketed ID String “<ID>”) then the padding base is required and POS denotes the coordinate of the base preceding the polymorphism. Tools processing VCF files are not required to preserve case in the allele Strings. (String, Required).